lib/components/Box/useBoxStyles.ts

```diff
@@ -48,10 +48,19 @@ interface Border {
 	borderRadius?: ResponsiveProp<keyof typeof styleRefs.borderRadius>;
 }
 
-export interface BoxStyleProps extends Padding, Margin, Border {
+interface Flex {
+	alignItems?: ResponsiveProp<keyof typeof styleRefs.alignItems>;
+	flexDirection?: ResponsiveProp<keyof typeof styleRefs.flexDirection>;
+	flexGrow?: keyof typeof styleRefs.flexGrow;
+	flexShrink?: keyof typeof styleRefs.flexShrink;
+	flexWrap?: keyof typeof styleRefs.flexWrap;
```

Comment:
Any chance we can make wrapping to also be responsive. I have had multiple use-cases for this I had to do as custom styles